### PR TITLE
Assignment1 

### PR DESCRIPTION
**Pull request by adityagandi1203**

### CODE REVIEW

File: src/HW1/FindGuitarTester.java [hunk 1]
```diff
@@ -1,4 +1,12 @@
...
+ * @01392295 Nikhil
+ * @01390645 Raghunandan
+ * 
+ * 
+ * 
+ * 
+ * */
 public class FindGuitarTester {
 
   public static void main(String[] args) {
```
Comment (omontalv) [inline src/HW1/FindGuitarTester.java]: Method comment?

File: src/HW1/FindGuitarTester.java [hunk 2]
```diff
@@ -1,4 +1,12 @@
 package HW1;
+/*01392300 Aditya
```
Comment (omontalv) [inline src/HW1/FindGuitarTester.java]: File header, class comment?

File: src/HW1/test/GuitarTest.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package HW1.test;
+package HW1;
```
Comment (omontalv) [inline src/HW1/test/GuitarTest.java]: I believe I wanted this in a different package.

**Comment by adityagandi1203** [comment]
Updated Assignment For given comments.

**Comment by adityagandi1203** [comment]
Assignment 1 Updated

**Pull request merged**